frontend/src/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,56 @@
+import './ProfilePage.scss';
+import React, {useContext, useEffect, useState} from "react";
+import BagPlace from "../models/BagPlace";
+import {getBagPlacesOfUser} from "../service/RequestService";
+import BagPlaceCounter from "../components/BagPlaceCounter";
+import {AuthContext} from "../context/AuthProvider";
+import {Button} from "@mui/material";
+import {useNavigate} from "react-router-dom";
+
+export default function ProfilePage() {
+
+    const [bagPlaces, setBagPlaces] = useState<BagPlace[]>([])
+
+    const {token} = useContext(AuthContext)
+
+    const navigate = useNavigate()
+
+    const setupBagPlaces = () => getBagPlacesOfUser(token).then(data => setBagPlaces(data))
```

Comment:
This is confusing. If the function is used only once there is no need to store it in a constant.

Reply:
Yeah, you're right.
I could just put my request inside my useEffect like this, couldn't I?
`useEffect(() => {
        getBagPlacesOfUser(token).then(data => setBagPlaces(data))
        //eslint-disable-next-line
    }, [])`

Reply:
yes